### PR TITLE
Feat/active projects view

### PR DESCRIPTION
**Pull request by LiWeiYeh**

### CODE REVIEW

File: src/app/project-administration/active-projects/active-projects.component.html [hunk 1]
```diff
@@ -29,13 +29,20 @@ <h2>Active Projects</h2>
...
             </td>
-            <td></td>
             <td>
                 <app-atomic-date [property]="row.Start" crud="cRUd"></app-atomic-date>
             </td>
-            <td></td>
+            <td>
+                <app-atomic-alphanumeric [property]="row.Status" label="Name" isUni isTot crud="CRUD">
+                </app-atomic-alphanumeric>
+            </td>
```
Comment (PatrickR079) [inline src/app/project-administration/active-projects/active-projects.component.html:45]: 1 <th> is missing, currently the table outline is not correct
image

File: src/app/shared/atomic-components/atomic-object/atomic-object.component.html [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
-        *ngIf="obj._ifcs_.length > 1"
-        label="{{ obj._label_ }}"
-        (onClick)="navigateToSingleEntity(obj._id_, obj._ifcs_[0])"
-        [model]="toPrimeNgMenuModel(obj._ifcs_)"
-        styleClass="p-button-text mr-2 mb-2"
-    ></p-splitButton>
+<p-splitButton
+    *ngIf="ifcs.length > 1"
+    label="{{ label }}"
+    (onClick)="navigateToEntity(ifcs[0].id, id)"
```
Comment (PatrickR079) [inline src/app/shared/atomic-components/atomic-object/atomic-object.component.html:4]: Throws an error if the path doesnt exist 
image

File: src/app/shared/atomic-components/atomic-object/atomic-object.component.ts [hunk 3]
```diff
@@ -8,41 +8,35 @@ import { Router } from '@angular/router';
...
-    {
-      label: 'Project',
-      routerLink: ['/project', '123'],
-      command: (event?) => {
-        console.log('click');
-      },
-    },
-  ];
+  public id!: string;
+  public ifcs!: Array<any>;
```
Comment (PatrickR079) [inline src/app/shared/atomic-components/atomic-object/atomic-object.component.ts:14]: no 'any' allowed

File: src/app/shared/atomic-components/atomic-object/atomic-object.component.ts [hunk 4]
```diff
@@ -8,41 +8,35 @@ import { Router } from '@angular/router';
...
-    return `/${ifc.id.toLowerCase()}`;
+  override ngOnInit(): void {
+    this.id = this.property._id_;
+    this.ifcs = this.property._ifcs_;
+    this.label = this.property._label_;
+    this.menu = this.toPrimeNgMenuModel(this.ifcs, this.id);
   }
 
-  toPrimeNgMenuModel(ifcs: Array<any>): Array<MenuItem> {
+  private toPrimeNgMenuModel(ifcs: Array<any>, id: string): Array<MenuItem> {
```
Comment (PatrickR079) [inline src/app/shared/atomic-components/atomic-object/atomic-object.component.ts:33]: no 'any' allowed

File: src/app/project-administration/active-projects/active-projects.component.html (hunk 1 above)
Comment (PatrickR079) [reply src/app/project-administration/active-projects/active-projects.component.html:45]: try using the "colspan" property

File: src/app/shared/atomic-components/atomic-object/atomic-object.component.html (hunk 2 above)
Comment (LiWeiYeh) [reply src/app/shared/atomic-components/atomic-object/atomic-object.component.html:4]: will fix in next PR.

File: src/app/shared/atomic-components/atomic-object/atomic-object.component.ts (hunk 3 above)
Comment (LiWeiYeh) [reply src/app/shared/atomic-components/atomic-object/atomic-object.component.ts:14]: there is already an issue open for anys

File: src/app/shared/atomic-components/atomic-object/atomic-object.component.ts (hunk 4 above)
Comment (LiWeiYeh) [reply src/app/shared/atomic-components/atomic-object/atomic-object.component.ts:33]: there is already an issue open for anys

File: src/app/project-administration/active-projects/active-projects.component.html (hunk 1 above)
Comment (LiWeiYeh) [reply src/app/project-administration/active-projects/active-projects.component.html:43]: made a separate item for this on the backlog

**Review by PatrickR079 — APPROVED** [review]

**Pull request merged**